src/features/unified-share-modal/flowTypes.js

```diff
@@ -68,6 +68,7 @@ export type SuggestedCollabLookup = {
 
 export type inviteePermissionType = {
     default: boolean,
+    description?: string,
```

Comment:
where ids this being used?

Reply:
used here: https://github.com/box/box-ui-elements/blob/ccec3770700703e5aea148ff2f491437e4c78d32/src/features/unified-share-modal/InviteePermissionsMenu.js#L61

`level` is type `inviteePermissionType`

Reply:
missed that, now i have mixed feelings on the fact that `customDescription` is being used in `InviteePermissionsDescription` instead of `description`

Reply:
I updated the prop name to just `description` renaming the previous description variable to `defaultDescription` in the `InviteePermissionDescription` component.

Regarding the `InviteePermissionsLabel` component, it already has an invitee related prop so I decided to stick to the similar name:

```
type Props = {
    hasDescription?: boolean,
    inviteePermissionDescription?: string,
    inviteePermissionLevel: string,
    itemType: ItemType,
};
```